CHANGELOG.md

```diff
@@ -26,7 +26,7 @@
 
   **Breaking change**
 
-  `TreeView.Item` and `TreeView.LinkItem` now require an `id` prop:
+  `TreeView.Item` now requires an `id` prop:
```

Comment:
We don't need to update previous changelog entries when we delete a component. We usually treat our changelog as immutable.